BootloaderCommonPkg/Include/Library/CryptoLib.h

```diff
@@ -31,7 +31,10 @@ typedef UINT8 KEY_TYPE;
 #define RSA2048_NUMBYTES              256
 #define RSA3072_NUMBYTES              384
 
-#define RSA_MOD_SIZE 256 //hardcode n size to be 256
+#define RSA2048_MOD_SIZE 256 //n size to be 256

```

Comment:
Let us align the names and values for easier read

Reply:
@mauricema were you suggesting to use RSA_MOD_SIZE_2048?

Reply:
no.   I meant the code format.  Put more spaces between RSA2048_MOD_SIZE and 256 so that numbers are all aligned like other defines.